test/Reader/IniTest.php

```diff
@@ -95,4 +95,26 @@ public function testFromStringNested()
         $this->assertEquals($arrayIni['bla']['foo']['baz'][0], 'foobaz1');
         $this->assertEquals($arrayIni['bla']['foo']['baz'][1], 'foobaz2');
     }
+
+    public function testFromFileParseSections()
+    {
+        $arrayIni = $this->reader->fromFile($this->getTestAssetPath('sections'));
+
+        $this->assertEquals($arrayIni['production']['env'], 'production');
+        $this->assertEquals($arrayIni['production']['production_key'], 'foo');
+        $this->assertEquals($arrayIni['staging : production']['env'], 'staging');
+        $this->assertEquals($arrayIni['staging : production']['staging_key'], 'bar');
```

Comment:
we should swap parameters order in assertions: 1st param is expected, 2nd actual

Reply:
Done